Documentation/README.md

```diff
@@ -1,4 +1,4 @@
-# AEP Analytics SDK Documentation
+# Adobe Analytics Extension Documentation
```

Comment:
Adobe Experience Platform - Analytics extension for Android

refer:https://github.com/adobe/aepsdk-media-android/tree/staging

Reply:
I followed Media to use "Analytics extension documentation".